### DISCUSSION TITLE
Sampling with uncertainty

### DISCUSSION THREAD

**Discussion by Mutaz94** [metrumresearchgroup/mrgsolve#862] [General]
Hi @kylebaron:

I'm curious about what methodology does mrgsolve follows when simulating with uncertainty. I know that `NONMEM` simulate using inverse Wishart distribution (conjugate prior to multivariate), others can use multivariate distributions (log or truncated) and stan does the LKJ strategy. Do we have the option to choose here using mrgsolve? 

Additional info: to make it clear, I mean by sampling, is simulating the full variance-covariance matrix. 

Thanks!

**Comment by kylebaron** [comment]
HI @Mutaz94 - 

mrgsolve only simulates ETA and EPS (both mutivariate normal); if you want to simulate OMEGA and SIGMA you could try this `simpar` package: https://github.com/metrumresearchgroup/simpar

Kyle

**Reply by Mutaz94** [reply]
Looks interesting, thanks!! But still `simpar` is using the multivariate normal distribution to simulate, right? You cannot use another conjugate distribution such as inverse Wishart or LKJ. Correct?

**Reply by curtisKJ** [reply]
IIRC `simpar` uses different distributions depending on the parameter type. Fixed effects are multivariate normal, random effects are either inverse Wishart or inverse Chi-squared.

**Reply by Mutaz94** [reply]
Make sense. Thanks!
